go.mod

```diff
@@ -52,11 +52,11 @@ require (
 	github.com/stretchr/testify v1.9.0
 	go.opencensus.io v0.24.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.31.0
+	golang.org/x/crypto v0.35.0
```

Comment:
https://github.com/golang/crypto/compare/v0.31.0...v0.35.0
blake2b only affected and it is x/sys version upgrade, looks ok